Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Dockerfile
+
+FROM phusion/passenger-ruby22:0.9.15
+
+# Set correct environment variables.
+ENV HOME /root
+ENV RAILS_ENV production
+
+# Use baseimage-docker's init process.
+CMD ["/sbin/my_init"]
+
+# Expose Nginx HTTP service
+EXPOSE 80
```

Comment:
It isn't clear to me that you can do it that way with elastic beanstalk.
This is the guide I remember - 
https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/SSLDocker.SingleInstance.html